docs/release-notes.rst

```diff
@@ -10,6 +10,26 @@
  Version 0.28
 **************
 
+Version 0.28.1
+==============
+
+**Release Date:** February 20, 2024
+
+**Improvements**
+
+-  GCS Storage client will now retry following the default policy on ``TooManyRequests`` rate limit
```

Comment:
```suggestion
-  The Google Cloud Storage client will now retry following the default policy on ``TooManyRequests`` rate limit
```

Reply:
good catch

Reply:
updated the usage guide https://hpe-aiatscale.atlassian.net/wiki/spaces/DOC/pages/1332707345/Usage+Guide